src/app/options/qgsvectorrenderingoptions.cpp

```diff
@@ -13,6 +13,7 @@
  *                                                                         *
  ***************************************************************************/
 
+#include "qgsabstractgeometry.h"
```

Comment:
Related?

Reply:
I cannot build without it, I have removed a few includes here and there so probably a side effect